Sources/PreviewGallery/ModuleFeatures.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ModuleFeatures.swift
+//  
+//
+//  Created by Noah Martin on 8/31/23.
+//
+
+import Foundation
+import SwiftUI
+
+struct ModuleFeatures: View {
+
+  let module: String
+  let data: Data
+
+  var body: some View {
+    let featureProviders = data.previews(in: module).filter { provider in
+      return provider.previews.contains { preview in
+        return preview.requiresFullScreen
+      }
+    }
+    return List {
+      ForEach(featureProviders) { provider in
+        let featurePreviews = provider.previews.filter { $0.requiresFullScreen }
+        NavigationLink {
+          if featurePreviews.count == 1 {
+            try! featurePreviews[0].view()
+          } else {
+            List {
```

Comment:
Nested lists work fine in SwiftUI? Worth testing and maybe converting to a regular VStack.